tracer/test/test-applications/integrations/Samples.InstrumentedTests/Vulnerabilities/String/StringConcatTests.cs

```diff
@@ -485,22 +496,22 @@ public void GivenATaintedObject_WhenCallingConcatWith4ObjectParams_ResultIsTaint
         AssertTaintedFormatWithOriginalCallCheck(":+-TAINTED2-+:concat:+-tainted-+:concat2", String.Concat(taintedValue2, (object)"concat", taintedValue, (object)"concat2"), () => String.Concat(taintedValue2, (object)"concat", taintedValue, (object)"concat2"));
     }
 
-    [Fact]
+    [Fact(Skip = "Aspect disabled until undefined generics are supported")]
```

Comment:
What is the issue with the undefined generics? Until now, they were supported, right? but there must be a strong reason to stop supporting them.

Reply:
They are not supported in instrumentation. We just added heuristic code to circumvent this, causing potential crashes. As the purpose of this PR is to add security to our code, these overloads were removed until a proper support is implemented.